File: critreel/critreel_codigo/critreel_ionic/projeto/src/app/model/critica.ts
export class Critica {
    idCritica: number;
    tituloFilme: string;
    texto: string;
    idUsuario: number;
    idFilme: number | null;
    dataCriacao: string;
    fotoCritica: string;

    constructor() {
        this.idCritica = 0;
        this.tituloFilme = '';
        this.texto = '';
        this.idUsuario = 0;
        this.idFilme = 0;
        this.dataCriacao = '';
        this.fotoCritica = '';
    }
}

File: critreel/critreel_codigo/critreel_api/critreelFinal/critreel/src/main/java/br/cefetmg/critreel/model/Critica.java
package br.cefetmg.critreel.model;

import lombok.*;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Critica {
    private Long idCritica;
    private String tituloFilme;
    private String texto;
    private Long idUsuario;
    private Long idFilme;
    private String dataCriacao;
    private String fotoCritica;
}
